incomplete type and constant declaration

The following program `sample.go` triggers a panic:

```go
package main

type T1 t1

type t1 int8

const (
    P2 T1 = 2
    P3 T1 = 3
)

func main() {
    println(P3)
}
```

Expected result:
```console
$ go run ~/yt/yo.go
3
```

Got:
```console
$ yaegi ~/yt/to.go
home/marc/yt/to.go::8:2: CFG post-order panic: /home/marc/yt/to.go:3:6: incomplete type T1
goroutine 1 [running]:
runtime/debug.Stack(0x1, 0xc00035e600, 0x40)
        /usr/lib/go/src/runtime/debug/stack.go:24 +0x9d
github.com/containous/yaegi/interp.(*Interpreter).Eval.func1(0xc000227de8)
        /home/marc/go/src/github.com/containous/yaegi/interp/interp.go:342 +0xbc
panic(0xd8a800, 0xc0002810c0)
        /usr/lib/go/src/runtime/panic.go:969 +0x166
github.com/containous/yaegi/interp.(*Interpreter).cfg.func2.1(0xc0003b2f00)
        /home/marc/go/src/github.com/containous/yaegi/interp/cfg.go:425 +0xb3
panic(0xd8a800, 0xc000281080)
        /usr/lib/go/src/runtime/panic.go:975 +0x3e3
github.com/containous/yaegi/interp.(*itype).frameType(0xc0003a45b0, 0xc000294ef0, 0xc0003a44e0)
        /home/marc/go/src/github.com/containous/yaegi/interp/type.go:1277 +0x1bc
github.com/containous/yaegi/interp.(*scope).add(0xc00039d1f0, 0xc0003a45b0, 0x2)
        /home/marc/go/src/github.com/containous/yaegi/interp/scope.go:172 +0x49
github.com/containous/yaegi/interp.(*Interpreter).cfg.func2(0xc0003b2f00)
        /home/marc/go/src/github.com/containous/yaegi/interp/cfg.go:484 +0x879b
github.com/containous/yaegi/interp.(*node).Walk(0xc0003b2f00, 0xc000227200, 0xc000227238)
        /home/marc/go/src/github.com/containous/yaegi/interp/interp.go:204 +0xaf
github.com/containous/yaegi/interp.(*node).Walk(0xc0003b2e00, 0xc000227200, 0xc000227238)
        /home/marc/go/src/github.com/containous/yaegi/interp/interp.go:201 +0x70
github.com/containous/yaegi/interp.(*Interpreter).cfg(0xc0002a21a0, 0xc0003b2e00, 0x7ffc13b1d5ac, 0x13, 0xc000227470, 0x540211, 0xc000282c00, 0x0, 0x0)
        /home/marc/go/src/github.com/containous/yaegi/interp/cfg.go:59 +0x223
github.com/containous/yaegi/interp.(*Interpreter).gta.func1(0xc0003b2e00, 0xc000227bb8)
        /home/marc/go/src/github.com/containous/yaegi/interp/gta.go:27 +0xae1
github.com/containous/yaegi/interp.(*node).Walk(0xc0003b2e00, 0xc000227bb8, 0x0)
        /home/marc/go/src/github.com/containous/yaegi/interp/interp.go:197 +0xc9
github.com/containous/yaegi/interp.(*node).Walk(0xc0003b2400, 0xc000227bb8, 0x0)
        /home/marc/go/src/github.com/containous/yaegi/interp/interp.go:201 +0x70
github.com/containous/yaegi/interp.(*Interpreter).gta(0xc0002a21a0, 0xc0003b2400, 0xc000294e90, 0x4, 0x7ffc13b1d5ac, 0x13, 0x0, 0xc00033e600, 0x0, 0x0, ...)
        /home/marc/go/src/github.com/containous/yaegi/interp/gta.go:19 +0x1e8
github.com/containous/yaegi/interp.(*Interpreter).gtaRetry(0xc0002a21a0, 0xc000227d90, 0x1, 0x1, 0xc000294e90, 0x4, 0x7ffc13b1d5ac, 0x13, 0x0, 0x0)
        /home/marc/go/src/github.com/containous/yaegi/interp/gta.go:322 +0x14a
github.com/containous/yaegi/interp.(*Interpreter).Eval(0xc0002a21a0, 0xc00034f280, 0x77, 0x0, 0x0, 0x0, 0x0, 0x0)
        /home/marc/go/src/github.com/containous/yaegi/interp/interp.go:364 +0x237
main.main()
        /home/marc/go/src/github.com/containous/yaegi/cmd/yaegi/yaegi.go:149 +0x5d8
```

--- a/_test/const15.go
+++ b/_test/const15.go
@@ -1,0 +1,17 @@
+package main
+
+type T1 t1
+
+type t1 int8
+
+const (
+	P2 T1 = 2
+	P3 T1 = 3
+)
+
+func main() {
+	println(P3)
+}
+
+// Output:
+// 3

--- a/interp/cfg.go
+++ b/interp/cfg.go
@@ -476,6 +476,9 @@ func (interp *Interpreter) cfg(root *node, pkgID string) ([]*node, error) {
 							dest.typ = sc.fixType(src.typ)
 						}
 					}
+					if dest.typ.incomplete {
+						return
+					}
 					if dest.typ.sizedef {
 						dest.typ.size = arrayTypeLen(src)
 						dest.typ.rtype = nil

